code/game/objects/items/food/lizard.dm

```diff
@@ -0,0 +1,564 @@
+	tastes = list("snails" = 1, "garlic" = 1, "oil" = 1)
+	foodtypes = MEAT | VEGETABLES
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/fried_blood_sausage
+	name = "fried blood sausage"
+	desc = "A blood sausage, battered and deep fried. Commonly served with fries as a quick and simple snack on the streets of Zagoskeld."
+	icon = 'icons/obj/food/lizard.dmi'
+	icon_state = "fried_blood_sausage"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 3, /datum/reagent/consumable/salt = 1, /datum/reagent/consumable/cooking_oil = 1)
+	tastes = list("black pudding" = 1, "batter" = 1, "oil" = 1)
+	foodtypes = MEAT | FRIED
+	w_class = WEIGHT_CLASS_SMALL
+
+//Why does like, every language on the planet besides English call them pommes? Who knows, who cares- the lizards call them it too, because funny.
+/obj/item/food/lizard_fries
+	name = "loaded poms-franzisks"
+	desc = "One of the many human foods to make its way to the lizards was french fries, which are called poms-franzisks in Draconic. When topped with barbecued meat and sauce, they make a hearty meal."
+	icon = 'icons/obj/food/lizard.dmi'
+	icon_state = "lizard_fries"
+	food_reagents = list(/datum/reagent/consumable/nutriment = 4, /datum/reagent/consumable/nutriment/protein = 3, /datum/reagent/consumable/salt = 1, /datum/reagent/consumable/bbqsauce = 2)
+	tastes = list("fries" = 2, "bbq sauce" = 1, "barbecued meat" = 1)
+	foodtypes = MEAT | VEGETABLES | FRIED
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/brain_pate
+	name = "eyeball-and-brain pate"
+	desc = "A thick pink puree made from finely chopped poached eyeballs and brains, fried onions, and fat. Lizards swear it's delicious!"
+	icon = 'icons/obj/food/lizard.dmi'
+	icon_state = "brain_pate"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 5)
+	tastes = list("brains" = 2)
+	foodtypes = MEAT | VEGETABLES | GROSS
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/crispy_headcheese
+	name = "crispy breaded headcheese"
+	desc = "A delicious snack from the streets of Zagoskeld, consisting of headcheese coated in rootbread breadcrumbs. Commonly served with fries."
+	icon = 'icons/obj/food/lizard.dmi'
+	icon_state = "crispy_headcheese"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 3, /datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/cooking_oil = 1)
+	tastes = list("cheese" = 1, "oil" = 1)
+	foodtypes = MEAT | VEGETABLES | NUTS | GROSS
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/kebab/picoss_skewers
+	name = "picoss skewer"
+	desc = "A popular Tiziran streetfood consisting of vinegar-marinated armorfish on a skewer with onion and chillis."
+	icon = 'icons/obj/food/lizard.dmi'
+	icon_state = "picoss_skewer"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 3, /datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/vinegar= 1, /datum/reagent/consumable/capsaicin = 1)
```

Comment:
Long enough that you can split this into multiple lines.

Reply:
I'm not saying I wouldn't do it- but I don't think any other food splits over more than one line. Would it not be better to just leave this as is?